### PR TITLE
Update dashboard panels border radius

### PR DESCRIPTION
**Pull request by marcoambrosini**

### CODE REVIEW

**Comment by ShGKme** [comment]
Changes dashboard widgets to be the same as main container in apps:

Before | After
---|---
image | image

File: apps/dashboard/src/DashboardApp.vue [hunk 1]
```diff
@@ -506,7 +506,7 @@ export default {
 	background-color: var(--color-main-background-blur);
 	-webkit-backdrop-filter: var(--filter-background-blur);
 	backdrop-filter: var(--filter-background-blur);
-	border-radius: var(--border-radius-rounded);
+	border-radius: var(--body-container-radius);
```
Comment (ShGKme) [inline apps/dashboard/src/DashboardApp.vue:509]: Probably, we should also add this variable to the theming.

**Review by ShGKme — APPROVED** [review]
Maybe also change the padding to be in line with border radius?

File: apps/dashboard/src/DashboardApp.vue (hunk 1 above)
Comment (marcoambrosini) [reply apps/dashboard/src/DashboardApp.vue:509]: Isn't this one already there?

**Comment by susnux** [comment]
I wonder why this is not styled in the [NcDashboardWidget](https://nextcloud-vue-components.netlify.app/#/Components/NcDashboard?id=ncdashboardwidget) :thinking: 

But you then also need to adjust all "guest-box" components (login etc) which we unified with the dashboard widgets not long ago

**Comment by marcoambrosini** [comment]
> I wonder why this is not styled in the [NcDashboardWidget](https://nextcloud-vue-components.netlify.app/#/Components/NcDashboard?id=ncdashboardwidget) 🤔

I also thought I would find it there 

> But you then also need to adjust all "guest-box" components (login etc)

Could you please point me to the components @susnux?

**Comment by marcoambrosini** [comment]
@susnux should we tackle this in separate prs so that we get the important stuff in?

File: apps/dashboard/src/DashboardApp.vue (hunk 1 above)
Comment (susnux) [reply apps/dashboard/src/DashboardApp.vue:509]: I think @ShGKme means adding it to theming / the theme from the theming app

**Review by susnux — APPROVED** [review]

**Pull request merged**